### PR TITLE
[SINGULAR] DDP-8404 Create a dialog for feature flags toggling

### PR DESCRIPTION
**Pull request by alextanlan**
How it works:

[screen-capture (5).webm](https://user-images.githubusercontent.com/7396837/186199006-d894f9bd-e339-4c72-8a5b-481dd13a1164.webm)

### CODE REVIEW

File: ddp-workspace/projects/ddp-singular/src/app/modules/material/material.module.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { MatExpansionModule } from '@angular/material/expansion';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
 import { MatButtonModule } from '@angular/material/button';
-
+import {MatSlideToggleModule} from '@angular/material/slide-toggle';
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-singular/src/app/modules/material/material.module.ts]: Need spaces inside braces. Style.

File: ddp-workspace/projects/ddp-singular/src/app/app.component.ts [hunk 2]
```diff
@@ -13,11 +15,16 @@ import { IGNORE_ANALYTICS_CLASS } from './constants/analytics';
...
 
   constructor(
     private elRef: ElementRef,
-    private analytics: AnalyticsEventsService
-  ) {}
+    private analytics: AnalyticsEventsService,
+    private dialog: MatDialog,
+    @Inject('ddp.config') private config: ConfigurationService
+  ) {
+    this.isProdMode = this.config.logLevel.toString() === '2';
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-singular/src/app/app.component.ts]: Let's just set it in pepperConfig.js . 
runtimeEnvironment = 'prod' | 'dev' | 'staging' | 'test'.
And let's pick it up via ConfigurationService the way we pickup the Auth0Domain or anything else that comes from pepperConfig.js.
You will have to modify ddp-angular/config/pepperConfig.js.ctmpl . Should not be too hard.

File: ddp-workspace/projects/ddp-singular/src/app/config/feature-flags/feature-flags.enum.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export enum FeatureFlagsEnum {
+  DDP_8404_Home_page_update = 'feature DDP_8404_Home_page_update',
+  DDP_8560_Dashboard_page_update = 'feature DDP_8560_Dashboard_page_update'
+}
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-singular/src/app/config/feature-flags/feature-flags.enum.ts:4]: Would suggest renaming to make it clearer what true or false flag would mean, e.g.: 
**Show**DDP8404HomePageUpdate

File: ddp-workspace/projects/ddp-singular/src/app/app.component.ts (hunk 2 above)
Comment (alextanlan) [reply ddp-workspace/projects/ddp-singular/src/app/app.component.ts]: Done.

File: ddp-workspace/projects/ddp-singular/src/app/modules/material/material.module.ts (hunk 1 above)
Comment (alextanlan) [reply ddp-workspace/projects/ddp-singular/src/app/modules/material/material.module.ts]: Fixed.

File: ddp-workspace/projects/ddp-singular/src/app/config/feature-flags/feature-flags.enum.ts (hunk 3 above)
Comment (alextanlan) [reply ddp-workspace/projects/ddp-singular/src/app/config/feature-flags/feature-flags.enum.ts:4]: Renamed.

**Review by juanseba21 — APPROVED** [review]

**Pull request merged**